src/utils/document-ready.ts

```diff
@@ -71,7 +70,7 @@ function onDocumentState(doc, condition, callback) {
  * @param {!Document} doc
  * @return {!Promise<!Document>}
```

Comment:
Non-blocking but we should probably just aim to remove all JSDoc annotations

Reply:
Good call, removed!